src/qibolab/transpilers/placer.py

```diff
@@ -0,0 +1,260 @@
+import random
+
+import networkx as nx
+from qibo.config import log, raise_error
+from qibo.models import Circuit
+
+from qibolab.transpilers.abstract import Placer, create_circuit_repr
+
+
+def assert_placement(circuit: Circuit, layout: dict, verbose=False) -> bool:
+    """Checks if layout is correct and matches the number of qubits of the circuit.
+
+    Args:
+        circuit (qibo.models.Circuit): Circuit model to check.
+        layout (dict): physical to logical qubit mapping.
+        verbose (bool): If ``True`` it prints info messages.
+
+    Returns ``True`` if the following conditions are satisfied:
+        - layout is written in the correct form.
+        - layout matches the number of qubits in the circuit.
+    otherwise returns ``False``.
```

Comment:
```suggestion
    Returns ``True`` if the following conditions are satisfied:
        - layout is written in the correct form.
        - layout matches the number of qubits in the circuit.
        
    otherwise returns ``False``.
```
For sphinx, lists end with a blank line. Otherwise it may incorrectly add it to the last list element.